### PR TITLE
Generate markdown

### PR DESCRIPTION
**Pull request by RunningLeon**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tests/regression/convert.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='From yaml export markdown')
+    parser.add_argument('yaml_cfg', help='yml config path')
+    parser.add_argument('output_md_file', help='Output markdown file path')
+    args = parser.parse_args()
+    return args
+
+
+def generate_inference_dict():
```
Comment (RunningLeon) [inline tests/regression/convert.py:21]: 将yaml传进这个函数吧， 而不是通过`args = parse_args()`来取

File: tests/regression/convert.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    inference_dict = {}
+    name_list = []
+    metafile_list = []
+    model_configs_list = []
+    piplines_list = []
+
+    with open(args.yaml_cfg, 'r') as f:
+        config = yaml.load(f, Loader=yaml.FullLoader)
+        config = edict(config)
+        for k, v in config.items():
```
Comment (RunningLeon) [inline tests/regression/convert.py:32]: 既然使用了easydict，可以直接遍历config.models呀

File: tests/regression/convert.py (hunk 1 above)
Comment (kaizhong2021) [reply tests/regression/convert.py:21]: 嗯嗯，好的老师，我去改一下～

File: tests/regression/convert.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            backend = get_backend(i['deploy_config'])
+            task_type = get_task_type(i['deploy_config'])
+
+            codebase_list.append(codebase)
+            backend_list.append(backend)
+            get_task_type_list.append(task_type)
+    return codebase_list, backend_list, get_task_type_list
+
+
+def website_list():
```
Comment (RunningLeon) [inline tests/regression/convert.py:73]: url前缀可以直接配置但每个项目的yaml文件里，如mmseg.yaml开头配置下

File: tests/regression/mmrotate.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 globals:
+  repo_url: https://github.com/open-mmlab/mmrotate/tree/master
```
Comment (RunningLeon) [inline tests/regression/mmrotate.yml:2]: the url not exists. `master` - > `main`

File: tests/regression/mmocr.yml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 globals:
+  repo_url: https://github.com/open-mmlab/mmocr/tree/master
```
Comment (RunningLeon) [inline tests/regression/mmocr.yml:2]: the url not exists. `master` - > `main`

**Pull request closed without merging**